spec/inputs/jdbc_spec.rb

```diff
@@ -5,10 +5,13 @@
 require "sequel/adapters/jdbc"
 require "timecop"
 require "stud/temporary"
+require "time"
+require "date"
 
 describe LogStash::Inputs::Jdbc do
+  ENV['TZ'] = "UTC"
```

Comment:
Since this affects the global state of the plugin runs and does not get cleaned up between individual spec tests... would it make more sense to use `stub_const` as such in the `:before` block?

```
stub_const('ENV', ENV.to_hash.merge('TZ' => 'UTC'))
```